### DISCUSSION TITLE
Marketing Team Weekly Meeting Planning 2022-03-30

### DISCUSSION THREAD

**Discussion by murillol** [WordPress/Marketing-Team#24] [Ideas]
Please add your suggestions for this week's Global Marketing Meeting on 30 March 2022.

[Agenda: Global Marketing Team, 30 March 2022]()

The meeting will be held in the Marketing channel on the Making WordPress Slack at 15:00 UTC. All welcome.

1. Welcome
2. Getting to know each other
    a. Attendees
    b. Social question of the week
3. [Notes from the Last Meeting]()
    a. Volunteer call for notetakers and for our bi-weekly coffee break.
4. Project Updates
    a. Social Media updates
    b. GitHub
    c. Updates from Make WordPress events
5. Diary/Calendar Dates
    a. Meeting time
    b. Follow this Google [Calendar with meetings, collaborations, and social activities](https://calendar.google.com/calendar/b/2?cid=NDQyN2g1cnZrN2EwYXA5Y3VpNDI5OGYzY3NAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). We also list important dates in this [Forthcoming Diary Dates](https://make.wordpress.org/marketing/2021/01/13/forthcoming-diary-dates/) post.
6. Open Floor

If you have any additions, please comment below.

**Comment by rmartinezduque** [comment]
Hello! I'd like to add an update: I'm helping Anjana with the next edition of the [Month in WordPress](https://wordpress.org/news/category/month-in-wordpress/) post, which aims to be published next Tuesday, April 5. If someone has a Make team update that would be valuable and relevant for the post, please feel free to share it on the Make Marketing Slack channel and tag `@Reyes Martínez` for awareness by Friday, April 1. I will also share the draft before publishing it on the Marketing channel. Thank you!

Note: if you would prefer to create a GitHub card for this, that's totally fine for me. Please let me know how I can help :)

**Reply by nalininsbs** [reply]
Link to a GitHub card for this edition for you. Please add links to it for the document. This card can be shared in Slack when you are ready. Thank you @rmartinezduque https://github.com/WordPress/Marketing-Team/issues/45

**Reply by rmartinezduque** [reply]
Thank you, @nalininsbs.
